docs/pages/deploy-a-cluster/helm-deployments/gcp.mdx

```diff
@@ -149,8 +149,7 @@ Go to the "Service Accounts" section of Google Cloud IAM & Admin.
 | storage-bucket-creator-role | Role you just created allowing creation of storage buckets |
 | dns-updater-role | Role you just created allowing updates to Cloud DNS records |
 | Cloud Datastore Owner | Grants permissions to create Cloud Datastore collections |
-| Storage Object Creator | Allows writing of Google Cloud storage objects |
-| Storage Object Viewer | Allows reading of Google Cloud storage objects |
+| Storage Object Admin | Allows read/write/delete of Google Cloud storage objects |
```

Comment:
The helm guide could probably use a fresh set of eyes as well - but it requires some extra permissions so for now I've opted to fix the most important error (lack of delete permissions).